nikola/plugins/task/indexes.py

```diff
@@ -117,6 +117,7 @@ def provide_context_and_uptodate(self, classification, lang, node=None):
             "title": self.site.config["INDEXES_TITLE"](lang) or self.site.config["BLOG_TITLE"](lang),
             "description": self.site.config["BLOG_DESCRIPTION"](lang),
             "pagekind": ["main_index", "index"],
+            "featured": [p for p in self.site.posts if p.post_status == 'featured' and lang in p.translated_to],
```

Comment:
How about untranslated featured posts, when `SHOW_UNTRANSLATED_POSTS` is set to `True`?

Reply:
Fair point, will do this in a while.